### PR TITLE
UI Access Levels

### PR DESCRIPTION
**Pull request by Vinnytile**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Oleh-Bodnar — APPROVED** [review]

File: src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx [hunk 1]
```diff
@@ -162,7 +166,7 @@ const AddAdministratorModal = (props: Props) => {
           <AutoComplete
             className="adminTypeSelect"
             options={[
-              { value: "Голова Станиці" },
+              { value: "Голова Станиці", disabled: activeUserRoles.includes("Заступник Голови Станиці") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx:169]: Wouldn't it be simplier to write disabled: activeUserRoles.includes ("Deputy Head of the Village") as "includes" returns true or false itself?

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx [hunk 2]
```diff
@@ -205,7 +209,7 @@ const AddCitiesNewSecretaryForm = (props: any) => {
         <AutoComplete
           className={classes.inputField}
           options={[
-            { value: "Голова Станиці" },
+            { value: "Голова Станиці", disabled: activeUserRoles.includes("Заступник Голови Станиці") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx:212]: the same here

File: src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.tsx [hunk 3]
```diff
@@ -202,7 +206,7 @@ const AddClubsNewSecretaryForm = (props: any) => {
         <AutoComplete
           className={classes.inputField}
           options={[
-            { value: "Голова Куреня" },
+            { value: "Голова Куреня", disabled: activeUserRoles.includes("Заступник Голови Куреня") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.tsx:209]: the same here

File: src/pages/Regions/AddRegionSecretaryForm.tsx [hunk 4]
```diff
@@ -153,7 +157,7 @@ const AddNewSecretaryForm = (props: any) => {
         <AutoComplete
           className={classes.inputField}
           options={[
-            { value: "Голова Округи" },
+            { value: "Голова Округи", disabled: activeUserRoles.includes("Заступник Голови Округи") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/Regions/AddRegionSecretaryForm.tsx:160]: the same here

File: src/pages/UserTable/AddRegionAdministratorForm.tsx [hunk 5]
```diff
@@ -125,7 +129,7 @@ const AddNewAdministratorForm = ({
             className={classes.inputField}
             onChange={handleClick}
             options={[
-              { value: "Голова Округи" },
+              { value: "Голова Округи", disabled: activeUserRoles.includes("Заступник Голови Округи") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/UserTable/AddRegionAdministratorForm.tsx:132]: the same here

File: src/pages/Club/AddAdministratorModal/AddAdministratorModal.tsx [hunk 6]
```diff
@@ -164,7 +168,7 @@ const AddAdministratorModal = (props: Props) => {
           <AutoComplete
             className="adminTypeSelect"
             options={[
-              { value: "Голова Куреня" },
+              { value: "Голова Куреня", disabled: activeUserRoles.includes("Заступник Голови Куреня") ? true : false },
```
Comment (tanyabiryuk) [inline src/pages/Club/AddAdministratorModal/AddAdministratorModal.tsx:171]: the same here

**Review by tanyabiryuk — CHANGES_REQUESTED** [review]

File: src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx (hunk 1 above)
Comment (Vinnytile) [reply src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx]: Done

File: src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.tsx (hunk 3 above)
Comment (Vinnytile) [reply src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.tsx]: Done

File: src/pages/Club/AddAdministratorModal/AddAdministratorModal.tsx (hunk 6 above)
Comment (Vinnytile) [reply src/pages/Club/AddAdministratorModal/AddAdministratorModal.tsx]: Done

File: src/pages/UserTable/AddRegionAdministratorForm.tsx (hunk 5 above)
Comment (Vinnytile) [reply src/pages/UserTable/AddRegionAdministratorForm.tsx]: Done

File: src/pages/Regions/AddRegionSecretaryForm.tsx (hunk 4 above)
Comment (Vinnytile) [reply src/pages/Regions/AddRegionSecretaryForm.tsx]: Done

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx (hunk 2 above)
Comment (Vinnytile) [reply src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx]: Done

**Review by tanyabiryuk — APPROVED** [review]

**Review by veronika-kolosova — APPROVED** [review]

**Pull request merged**